src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -396,6 +397,10 @@ public class FlowStateAllowedEventService {
             PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT.fullName(),
             List.of(TAKE_CASE_OFFLINE)
         ),
+        entry(
```

Comment:
prefer to add unit tests.

Reply:
there are already test case written for this new predicate